spec/models/merchant_spec.rb

```diff
@@ -15,41 +15,61 @@
 
   describe 'instance methods' do
     before(:each) do
+      @merchant_user_1 = User.create!(name: 'Meg',
+                                   street_address: '123 Stang Ave',
+                                   city: 'Hershey',
+                                   state: 'PA',
+                                   zip_code: 17033,
+                                   email: 'roman@example.com',
+                                   password: 'hamburger01',
+                                   role: 1)
+      @merchant_user_2 = User.create!(name: 'Brian',
+                                   street_address: '123 Brian Ave',
+                                   city: 'Denver',
+                                   state: 'CO',
+                                   zip_code: 17033,
+                                   email: 'roman@examples.com',
+                                   password: 'hamburger02',
+                                   role: 1)
+      @merchant_user_3 = User.create!(name: 'Dao',
+                                   street_address: '123 Mike Ave',
+                                   city: 'Denver',
+                                   state: 'CO',
+                                   zip_code: 17033,
+                                   email: 'roman@exampless.com',
+                                   password: 'hamburger03',
+                                   role: 1)
+
       @meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      @chain = @meg.items.create(name: "Chain", description: "It'll never break!", price: 40, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 22)
       @tire = @meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
     end
     it 'no_orders' do
       expect(@meg.no_orders?).to eq(true)
 
-      order_1 = Order.create!(name: 'Meg', address: '123 Stang Ave', city: 'Hershey', state: 'PA', zip: 17033)
-      item_order_1 = order_1.item_orders.create!(item: @tire, price: @tire.price, quantity: 2)
+      order_1 = @merchant_user_1.orders.create!(name: 'Meg', address: '123 Stang Ave', city: 'Hershey', state: 'PA', zip: 17033)
```

Comment:
This translates very accurately what was in the original test. The original test is a bit confusing though (it reads like Meg ordering from herself which technically is possible but a bit odd)